basil/HL/SensirionBridgeDevice.py

```diff
@@ -0,0 +1,53 @@
+# ------------------------------------------------------------
+# Copyright (c) All rights reserved
+# SiLab, Institute of Physics, University of Bonn
+# ------------------------------------------------------------
+#
+
+from basil.HL.RegisterHardwareLayer import HardwareLayer
+
+class SensirionBridgeI2CDevice(HardwareLayer):
```

Comment:
Is this additional HL actually needed? As far as i can tell, the commands here control the connected sensor (directly or indirectly) and could be integrated in the corresponding HL. I'm not sure we will ever use something other than the T / H  sensors we already have with this bridge...

Reply:
I just thought, that this way more sensors could be added. But if we will not use any other sensors, having only the one file would be simpler.

Reply:
mhh.. what other sensors does this bridge support at all? sht35 is basically the same? (apart from the heater, obviously).. maybe it makes sense to also support this. the sht35 that we have from the old ek-h4 should work with this bridge as well, correct?

Reply:
No it does not. It has its own protocol, that is simillar to I2C but not suported by the bridge. The bridge can be used to communicate arbitrary I2C or SPI devices.

Reply:
ok, i guess in this case the general implementation makes sense. but that's a TL in the strict definition. why do you need a standalone HL?

Reply:
My thought behind this seperation was: 
- The bridge enables cummunication to devices of multible types (currently just I2C). So its defined in the TL.
- Multiple devices can be connected to the bridge. They are defined in the HL.
- Information like the bridge port or the I2C adress have to be stored per device and are therefore also part of the HL. But since they are not specific to the device type, I defined a generic I2C device, that stores this kind of information and handels all that is specific to the bridge.
- A specific device inherits from this class and can just use the more generic functions without having to know how the bridge works

I hope this makes sense

Reply:
But you are right. Technicly this should be TL, but than we would need to have two TL, one for the bridge and anover for each device. I'm not sure, how to implement this correctly.

Reply:
i see. another layer of base class.. ok, i guess. if there's enough generic stuff to justify this, why not.